web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -253,3 +253,28 @@ WHERE {
 QUERY;
   $connection->query($query);
 }
+
+/**
+ * Force-update all distribution aliases.
+ */
+function joinup_core_update_8113(&$sandbox) {
+  if (!isset($sandbox['entity_ids'])) {
+    $sandbox['entity_ids'] = \Drupal::entityQuery('rdf_entity')
+      ->condition('rid', 'asset_distribution')
+      ->execute();
+    $sandbox['current'] = 0;
+    $sandbox['max'] = count($sandbox['entity_ids']);
+  }
+
+  $limit = 50;
+  $entity_storage = \Drupal::entityTypeManager()->getStorage('rdf_entity');
+  $result = array_slice($sandbox['entity_ids'], $sandbox['current'], $limit);
+  foreach ($result as $entity_id) {
+    $entity = $entity_storage->load($entity_id);
+    \Drupal::service('pathauto.generator')->updateEntityAlias($entity, 'update', ['force' => TRUE]);
```

Comment:
Why are we computing this service every single iteration? Shouldn't we assign it to a variable outside `foreach ()`?